doc/engsys_resource_management.md

```diff
@@ -0,0 +1,107 @@
+## Resource Management Guidelines
+
+This document contains guidelines for creating and managing Azure resources in the Azure SDK Engineering System
```

Comment:
```suggestion
This document contains guidelines for creating and managing Azure resources in the Azure SDK
```
I don't want to confuse things with "Engineering System" as that is a particular subscription.